openerp/models.py

```diff
@@ -60,6 +60,7 @@
 
 regex_order = re.compile('^(\s*([a-z0-9:_]+|"[a-z0-9:_]+")(\s+(desc|asc))?\s*(,|$))+(?<!,)$', re.I)
 regex_object_name = re.compile(r'^[a-z0-9_.]+$')
+regex_pg_name = re.compile(r'[a-z_][a-z0-9_$]*', re.I)
```

Comment:
Shouldn't it be `re.compile(r'^[a-z_][a-z0-9_$]*$', re.I)`?
It seems to me that `regex_pg_name.match('abc!') returns a group with 'abc' with the current regex.